src/C++/FieldConvertors.cpp

```diff
@@ -65,10 +65,10 @@ namespace FIX
 	  return g_atod_converter.StringToDouble( buffer, size, processed_chars );
   }
 
-  int DoubleConvertor::fast_dtoa( char * buffer, int size, double value )
+  int DoubleConvertor::fast_dtoa( char * buffer, int size, double value, int significant_digits )
```

Comment:
Updated to allow for user to set the desired significant digits instead of the hard coded constant value.  Hard coded constant value prevented an 'if' block branch from being hit due to the hard coded value never being able to return a false.